examples/autoscaler-webhook/Dockerfile

```
@@ -22,7 +22,7 @@ RUN go mod init && go mod tidy
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o server .
 
 # Create the final image that will run the webhook server for FleetAutoscaler webhook policy
-FROM alpine:3.14
+FROM gcr.io/distroless/static-debian11:nonroot
 RUN adduser -D -u 1000 server
```

```suggestion
WORKDIR /
```

No need for this anymore, and we move everything into root.